Makefile.am

```diff
@@ -60,20 +60,22 @@ PROFILE_DEFAULT=--profile=log:time=fast,calls,noalloc,maxframes=5
 PROFILE_HEAP=--profile=logging:heap=all,gc-dumps=-1
 PROFILE_GUI=--profile=gui-thread-check
 
+RUN_DEFAULT=--debug --uninstalled
```

Comment:
I actually wanted to get `$*` into here, IIRC I tried `RUN_DEFAULT='--debug --uninstalled $*'` at the time but it didn't work, although I don't recall what was passed through, worth a second look